docs/nextflow/use-cases.md

```diff
@@ -16,234 +17,262 @@ To get started with an example Nextflow pipeline that uses Wave, see [Nextflow a
 
-To provision container images on demand, add the Dockerfile of the container to the [module directory](https://www.nextflow.io/docs/latest/module.html#module-directory) where the pipeline process is defined.
-When Wave is enabled, it automatically uses the Dockerfile to build the required container, uploads the container to the registry, and uses the container to execute the script defined in the process.
+To enable Wave to build Nextflow module containers:
 
-:::note
-If a process declares a `container` directive, it takes precedence over the Dockerfile definition.
-:::
+1. Add your Dockerfile to the [module directory](https://www.nextflow.io/docs/latest/module.html#module-directory) where you define the pipeline process.
+
+1. Enable Wave in your Nextflow configuration:
+
+    ```groovy
+    wave.enabled = true
+    ```
 
-To use the Dockerfile in the module directory, even when a process uses a `container` directive, add the following to your Nextflow configuration file:
+1. (Optional) Set your Wave strategy to prioritize Dockerfiles in your Nextflow configuration:
 
-```groovy
-wave.strategy = ['dockerfile','container']
-```
+    ```groovy
+    wave.strategy = ['dockerfile','container']
+    ```
+
+    :::note
+    The `container` directive takes precedence over a Dockerfile by default.
+    :::
 
 :::warning
 Wave does not support `ADD`, `COPY`, or other Dockerfile commands that access files in the host file system.
 :::
 
 </details>
 
-### Build Conda-based containers
+## Build Conda-based containers
 
 Wave can provision containers based on the [`conda` directive](https://www.nextflow.io/docs/latest/process.html#conda).
 This allows you to use Conda packages in your pipeline, even on cloud-native platforms like AWS Batch and Kubernetes, which do not support the Conda package manager directly.
 
 <details open>
 <summary>**Build Conda-based containers**</summary>
 
-Define the `conda` requirements in your pipeline processes.
-Ensure the process doesn't include a `container` directive or Dockerfile.
+To enable Wave to provision Conda package containers:
+
+1. Define your Conda packages using the `conda` directive in your pipeline processes.
 
-To prioritize `conda` over `container` directives and Dockerfiles, add the following to your Nextflow configuration:
+1. Enable Wave in your Nextflow configuration:
 
-```groovy
-wave.strategy = ['conda']
-```
+    ```groovy
+    wave.enabled = true
+    ```
 
-For Nextflow 23.10.0, or later, the `conda-forge::procps-ng` package is automatically included in provisioned containers. This package includes the `ps` command.
+1. (Optional) Set your Wave strategy to prioritize `conda` in your Nextflow configuration:
 
-You can set Conda channels and their priority with `conda.channels`:
+    ```groovy
+    wave.strategy = ['conda']
+    ```
 
-```groovy
-wave.strategy = ['conda']
-conda.channels = 'seqera,conda-forge,bioconda,defaults'
-```
+    :::note
```

Comment:
The back-to-back :::note admonitions won't look great. Can you either combine them into one with bullets, or make one of them an :::info ?

Reply:
I didn't feel good about this either. Making the second point an `info`